packages/core/src/DirectoryLoader.ts

```diff
@@ -54,6 +54,48 @@ export abstract class DirectoryLoader {
 		return path.resolve(this.directory, file);
 	}
 
+	/**
+	 * Whether any of the node's credential types may be used to
+	 * make a request from a node other than itself.
+	 */
+	private supportsProxyAuth(description: INodeTypeDescription) {
+		if (!description.credentials) return false;
+
+		return description.credentials.some(({ name }) => {
+			const credType = this.credentialTypes[name].type;
```

Comment:
This check here assumes that all credentials are loaded before the nodes. This is breaking custom nodes. [Here](https://github.com/n8n-io/n8n/pull/5344) is the fix.